kbs/hack/ci-helper.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Is this `Intel` mark intended?

Reply:
it's copied from CAA, which copied it from kata afaik, so I left the header in